package mpsqual;

message SystemMonitoringRequest { 
}

message SystemMonitoringResponse {
	message SensorValue {	
			required string deviceName = 1; 
			required string sensorName = 2;
			required string value = 3;
	}

	message SemaValue {
		required string itemName = 1;
		required string value = 2;
	}

	message SwitchData {
		required string temperture = 1;
		repeated string statistics = 2;
	}

    repeated SensorValue powerSupplyStatistics = 1;
	repeated SemaValue semaStatistics = 2;
    required SwitchData	switchData = 3;
}
